src/eradiate/_config.py

```diff
@@ -87,6 +87,12 @@ def _data_path_validator(self, var, paths):
         help="Path to the Eradiate download directory.",
     )
 
+    systemtest_output_dir = environ.var(
+        default="$ERADIATE_DIR/test_report/generated",
+        converter=lambda x: pathlib.Path(os.path.expandvars(x)).absolute(),
+        help="Path to the Eradiate system test plot outputs.",
+    )
+
```

Comment:
Test artifact output has little to do with Eradiate per se. I'd prefer that this configuration would be scoped to tests only. I'm sure pytest has infrastructure for passing configuration items upon launching the test session --- something like what you did for JSON artifacts. Can you investigate that?

Reply:
[This](https://stackoverflow.com/questions/40880259/how-to-pass-arguments-in-pytest-by-command-line) seems like a good way to start.